BunkumGeminiSampleApplication/Endpoints/WeatherEndpoints.cs

```diff
@@ -0,0 +1,61 @@
+using Bunkum.Core;
+using Bunkum.Core.Database.Dummy;
+using Bunkum.Core.Endpoints;
+using Bunkum.Core.Storage;
+using Bunkum.Listener.Protocol;
+using Bunkum.Protocols.Gemini;
+using BunkumGeminiSampleApplication.Configuration;
+using BunkumGeminiSampleApplication.Models;
+
+namespace BunkumGeminiSampleApplication.Endpoints;
+
+public class WeatherEndpoints : EndpointGroup
+{
+    [GeminiEndpoint("/api/v1/weather", ContentType.Json)]
```

Comment:
Is JSON valid for Gemini?

Reply:
Yes, any MIME type is acceptable to be sent over the protocol